cypress/e2e/shared/firstmile.tests.ts

```diff
@@ -21,64 +20,64 @@ describe('First mile experience', () => {
       cy.contains('Hello, Time-Series World!')
 
       // second page is install dependencies
-      cy.contains('Next').click()
+      cy.getByTestID('python-next-button').click()
       cy.contains('Install Dependencies')
 
       // third page is create token
-      cy.contains('Next').click()
-      cy.contains('Create a Token')
+      cy.getByTestID('python-next-button').click()
+      cy.contains('Tokens')
 
       // fourth page is initalize client
-      cy.contains('Next').click()
-      cy.contains('Initalize Client')
+      cy.getByTestID('python-next-button').click()
+      cy.contains('Initialize Client')
 
       // fifth page is write data
-      cy.contains('Next').click()
+      cy.getByTestID('python-next-button').click()
       cy.contains('Write Data')
 
       // sixth page is execute query
-      cy.contains('Next').click()
+      cy.getByTestID('python-next-button').click()
       cy.contains('Execute a Flux Query')
 
       // seventh page is execute aggregate query
-      cy.contains('Next').click()
+      cy.getByTestID('python-next-button').click()
       cy.contains('Execute an Aggregate Query')
 
       // eight page is finish
-      cy.contains('Next').click()
+      cy.getByTestID('python-next-button').click()
       cy.contains('Congrats!')
     })
 
     describe('Subway Nav Bar', () => {
       it('navigates to the correct page when the respective navigation button is clicked', function() {
         cy.get('h1').contains('Hello, Time-Series World!')
 
-        cy.contains('Previous').should('be.disabled')
+        cy.getByTestID('python-prev-button').should('be.disabled')
 
-        cy.contains('Install Dependencies').click()
+        cy.contains(/^Install Dependencies$/).click()
```

Comment:
Do you know why searching for the string 'Install Dependencies' is flaky but searching for the regex `^Install Dependencies$`?

Putting `^Some Text$` into https://regex101.com/ (a great tool if you're not familiar) tells us:

```
^ asserts position at start of a line
Some Text
 matches the characters Some Text literally (case sensitive)
$ asserts position at the end of a line
```

Do you know why the text without `^` and `$` flakes out?

Reply:
Again something similar happened to above. When clicking on the `Tokens` step in the python wizard, for some reason, using `cy.contains('Tokens')` wanted to click on `API Tokens` in the other nav bar. So changing it to regex fixed that one, but I changed that on all of them so that anything similar to that wouldn't happen again.

Reply:
Good thinking, ole boy!